styles.css

```diff
@@ -1 +1,84 @@
-/*Maak hier jouw CSS!*/
\ No newline at end of file
+/*Maak hier jouw CSS!*/
+
+body {
+    background-color: #FCEFDD;
+}
+
+div {
+    min-width: 576px;
+    background-color: #FCEFDD;
+}
+
+.container {
+    background-color: #FCEFDD;
+    display: grid;
+    grid-gap: 16px;
+    grid-template-columns: 1fr;
+    grid-template-rows: 50px auto 50px 50px 50px 50px;
+    grid-template-areas:
+            "A"
+            "B"
+            "C"
+            "D"
+            "E"
+            "Z"
+            "F";
+}
+
+.grid-item {
+    background-color: #AC4449;
+    color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    border-style: solid;
+    border-color: #88363A;
+    padding: 10px;
+}
+
+.header {
+    grid-area: A;
+}
+
+.main {
+    grid-area: B;
+}
+
+.article {
+    grid-area: C;
+}
+
+.aside-small {
+    grid-area: D;
+}
+
+.aside-big {
+    grid-area: E;
+}
+
+.empty {
```

Comment:
Foei :-D deze is erg verkeerd. Deze hoeft NIET een eigen CSS of HTML element te krijgen - want in Grid kun je empty areas maken met een puntje in het template.